tracer/test/snapshots/Security.AspNetMvc5ApiSecurity.enableApiSecurity=False.__scenario=scan-without-attack.verified.txt

```diff
@@ -0,0 +1,57 @@
+[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet-mvc.request,
+    Resource: POST /home/apisecurity/{id},
+    Service: sample,
+    Type: web,
+    ParentId: Id_3,
+    Tags: {
+      aspnet.action: apisecurity,
+      aspnet.controller: home,
+      aspnet.route: {controller}/{action}/{id},
+      env: integration_tests,
+      http.client_ip: 127.0.0.1,
+      http.method: POST,
+      http.request.headers.host: localhost:00000,
+      http.status_code: 200,
+      http.url: http://localhost:00000/home/apisecurity/12,
+      http.useragent: Mistake Not...,
+      language: dotnet,
+      network.client.ip: ::1,
+      span.kind: server
+    }
+  },
+  {
+    TraceId: Id_1,
+    SpanId: Id_3,
+    Name: aspnet.request,
+    Resource: POST /home/apisecurity/{id},
+    Service: sample,
+    Type: web,
+    Tags: {
+      env: integration_tests,
+      http.client_ip: 127.0.0.1,
+      http.method: POST,
+      http.request.headers.host: localhost:00000,
+      http.route: {controller}/{action}/{id},
+      http.status_code: 200,
+      http.url: http://localhost:00000/home/apisecurity/12,
+      http.useragent: Mistake Not...,
+      language: dotnet,
+      network.client.ip: ::1,
+      runtime-id: Guid_1,
+      span.kind: server,
+      _dd.runtime_family: dotnet
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.agent_psr: 1.0,
+      _dd.appsec.enabled: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 2.0
```

Comment:
Since this is not an attack, should this be `_sampling_priority_v1: 1.0`?

Reply:
thanks a lot for this good catch, it was because the waf initialization sets priority to 2, so fixed in 89ea88284ff6917c5d14079fb17981d5fef7e7c3 with a first blank request